c2corg_ui/static/js/maincontroller.js

```diff
@@ -69,3 +69,10 @@ app.MainController.prototype.isPath = function(path) {
   }
 };
 
+
+/**
+ * @export
+ */
+app.MainController.prototype.animateHeaderIcon = function(e) {
+  app.utils.animateHeaderIcon(e);
```

Comment:
Can't you directly use ``app.utils.animateHeaderIcon(e);`` in the HTML instead of this alias in the main controller?